java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -50,6 +50,53 @@ protected ColumnView(long address) {
     this.nullCount = ColumnView.getNativeNullCount(viewHandle);
   }
 
+  /**
+   * Create a new column vector based off of data already on the device.
```

Comment:
This is a view, not a vector, and the comment needs to emphasize that the view is only valid as long as _all_ of the memory buffers being passed to this constructor are valid.  It does not change reference counts nor close these buffers when the view is closed.

Same comment applies to the other constructors.

Reply:
I think you caught this right before my push to clean this up.